internal/provider/clickhouse_cluster_resource_test.go

```diff
@@ -27,6 +62,7 @@ func TestAccClickhouseClusterResource(t *testing.T) {
 		RegionId:  types.StringValue("eu-central-1"),
 		CloudType: types.StringValue("aws"),
 		NetworkId: types.StringValue(testNetworkId),
+		Version:   types.StringValue("24.8"),
```

Comment:
We especially didn't fill the version here to have opportunity to run these tests on current version.
Does it work only on 24.8?

Reply:
On 24.7+